server/src/app.dtos.ts

```diff
@@ -1,23 +1,24 @@
 
 
-type QID = string;
-type AID = string;
-type UID = string;
-type Time = string; // subject to change
-type ResponsesLeft = number;
-
-export interface Question {
-  qid: QID, 
-  question: string, 
+//? to export or not to export
+export type QID = string;
+export type AID = string;
+export type UID = string;
+export type Time = string; // subject to change
+export type ResponsesLeft = number;
```

Comment:
Do these need to be exported? I don't see them being used anywhere

Reply:
I'm using them in app.service. 

Since APIAnswer and APIQuestion use these types, they should have the same scope, or at least the ability to be imported. It may be better to place in a different file